client/src/main/java/model/FleetCreator.java

```diff
@@ -0,0 +1,59 @@
+package model;
+
+import gui.fields.Mast;
+
+import java.util.ArrayList;
+import java.util.List;
+/**
+ * It holds information regarding a player's fleet.
+ *
+ * @author Patrycja Mikulska
+ * @version 1.5
+ */
+
+public class FleetCreator {
```

Comment:
Please write tests for this class

Reply:
I've changed this class as I could. I don't now how to test it :( so probably this class is bad.
Can You give me some suggestions how to test it or how to refactor to it to be testable?